common/reducers/swap.js

```diff
@@ -0,0 +1,60 @@
+import {
+    SWAP_DESTINATION_AMOUNT,
+    SWAP_DESTINATION_KIND,
+    SWAP_ORIGIN_AMOUNT,
+    SWAP_ORIGIN_KIND,
+    SWAP_UPDATE_BITY_RATES
+} from 'actions/swap';
+
+import {without} from 'lodash';
```

Comment:
`import without from lodash/without` would be easier to bundle w/o all other methods